### ISSUE TITLE
i dont know where this error message is coming from

### ISSUE THREAD

**Issue by codeincraft** [londonappbrewery/flash-chat-flutter#38]
![IMG-20210526-WA0037](https://user-images.githubusercontent.com/80710391/119586222-2f66c200-bd57-11eb-9abc-09c65423d399.jpeg)
![IMG-20210526-WA0039](https://user-images.githubusercontent.com/80710391/119586227-32fa4900-bd57-11eb-9d56-deafb4d073c1.jpeg)
![IMG-20210526-WA0041](https://user-images.githubusercontent.com/80710391/119586234-34c40c80-bd57-11eb-9def-0ebcca801c3d.jpeg)
![Screenshot_20210526-011357](https://user-images.githubusercontent.com/80710391/119586240-37befd00-bd57-11eb-8cd3-a808d1daa60e.png)

**Comment by eatmyaspirin** [comment]
from what I can see apparently the text widget you use in your "messagebubbles" isnt actually getting any data ie. the string variable is null
from what I can see, you did 
`final messageText = message.data['text'];`
`final messageSender = message.data['sender'];`
using the .get method on the "message" object should fix it :
`final messageText = message.get('text');`
`final messageSender = message.get('sender');`

alternatively
`final messageText = message['text'];`
`final messageSender = message['sender'];`

**Comment by codeincraft** [comment]
yeah, thanks to you all for your comments, actually I already fixed the problem. it was actually coming from my firebase set up. I was passing a null argument into my fire base
